libs/ui/editor/src/lib/blocks/blockMap.tsx

```diff
@@ -39,6 +39,8 @@ import {EditorBlockType, TeaserListBlockSort, TeaserType} from '@wepublish/edito
 import {isFunctionalUpdate} from '../utility'
 import {TeaserListBlock} from './teaserListBlock'
 import {SubscribeBlock} from './subscribeBlock'
+import {TeaserSlotsBlock} from './teaserSlotsBlock'
+import {TeaserSlotType} from '@wepublish/block-content/api'
```

Comment:
Importing API here breaks the build. should be imported from `@wepublish/editor/api-v2`